Sources/StreamChatUI/ChatMessageList/RETHINK/MessageContentView.swift

```diff
@@ -293,6 +292,14 @@ class MessageContentView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
                 )
             ]
         }
+
+        constraintsToActivate += fullWidthViews.map { view in
+            view.widthAnchor
+                .pin(equalTo: mainContainer.widthAnchor)
+                // should work with just .streamLow but it doesn't
+                // work for stream preview
+                .with(priority: .streamAlmostRequire)
+        }
```

Comment:
That's probably the most interesting part of the PR but not sure `maxWidth` option is valuable for the new attachments approach